lib/rubocop/config_loader.rb

```diff
@@ -139,7 +139,10 @@ def add_excludes_from_files(config, config_file)
         exclusion_file = find_last_file_upwards(DOTFILE, config_file, ConfigFinder.project_root)
```

Comment:
Actual fix needs to modify find_last_file_upwards to not return the home .yml file, but whatever config it finds before that home dotfile.